api/setup.js

```diff
@@ -32,8 +35,13 @@ if (options.circuitSize !== undefined) {
 // Log script name
 console.log(">> SET UP", "\n");
 
+const dirname = path.dirname(fileURLToPath(import.meta.url));
+const wasm = fs.readFileSync(path.join(dirname, "../", wasmPath));
+const wasmUnit8Array = new Uint8Array(wasm);
+
 let { md5, taskId, success } = await zkgapi.setup(
-  wasmPath,
+  "poc.wasm",
```

Comment:
what's this for? why use "write to file then read" instead of wrapping a File object with wasmUnit8Array?

Reply:
ok, should be an item for zkgraph-api to take care